src/nvim/ui.c

```diff
@@ -292,9 +293,13 @@ void ui_attach_impl(UI *ui)
     ui_send_all_hls(ui);
   }
   ui_refresh();
+
+  if (!comp) {
```

Comment:
@bfredl is there a more clever way to check if the UI is the compositor? `ui_comp_init` happens before eval is available, also I guess we don't want an extra UIAttach event just for the compositor. 

see also https://github.com/neovim/neovim/issues/10937#issuecomment-528236541

Reply:
The first UI that gets added `uis[0]` is always the compositor.